src/coreclr/gc/gcpriv.h

```diff
@@ -5694,6 +5696,12 @@ class region_allocator
     size_t get_free() { return (total_free_units * region_alignment) ; }
     size_t get_region_alignment () { return region_alignment; }
     size_t get_large_region_alignment () { return large_region_alignment; }
+    size_t get_used_region_count()
+    {
+        return ((region_map_right_start == region_map_right_end)
+              ? (region_map_left_end  - region_map_left_start)
+              : (region_map_right_end - region_map_left_start));
```

Comment:
actually I do think we don't need to care about the region_map_right case... since it's not used at all and the way we were thinking of using it will make this calculation incorrect anyway. so for now we can assume we don't need to worry about it at all.

Reply:
Ok, perhaps put an assert instead of the ?:

Reply:
sounds good to me.